docs/source/best_practices.rst

```diff
@@ -44,12 +62,21 @@ No::
 Ergonomic Practices
 ===================
 
-Using `Any`
------------
+Using ``Any`` and ``object``
+----------------------------
 
 Generally, use ``Any`` when a type cannot be expressed appropriately
 with the current type system or using the correct type is unergonomic.
 
+If a function accepts virtually any object as an argument, for example
```

Comment:
```suggestion
If a function accepts every possible object as an argument, for example
```
Avoid using the word "any" to describe how to use "object", it's going to be confusing.